app/workers/tweet_worker.rb

```diff
@@ -44,4 +44,23 @@ def text
     end
     clean_text
  end
+
+ def status
+    if @tweet.attrs[:retweeted]
+      "retweet"
+    elsif @tweet.attrs[:is_quote_status]
+      "quote"
+    else
+      "normal"
+    end
+  end
+
+  def user
+    user = if status == "retweet"
+             @tweet.attrs[:retweeted_status][:user]
+           else
+             @tweet.attrs[:user]
+    end
+    UserQuery.new(user[:name], user[:screen_name], user[:profile_image_url].remove("_normal"))
```

Comment:
What does `remove("_normal")` do here?

Ah ok. Just figured this out. This is a great instance where naming this would be really helpful:
```
large_profile_image_url = user[:profile_image_url].remove("_normal"))
```